controllers/extension-certificate-service/README.md

```diff
@@ -0,0 +1,103 @@
+# [Gardener Extension for certificate services](https://gardener.cloud)
+
+[![Go Report Card](https://goreportcard.com/badge/github.com/gardener/gardener-extensions/controllers/extension-certificate-service)](https://goreportcard.com/report/github.com/gardener/gardener-extensions/controllers/extension-certificate-service)
+
+Project Gardener implements the automated management and operation of [Kubernetes](https://kubernetes.io/) clusters as a service. Its main principle is to leverage Kubernetes concepts for all of its tasks.
+
+Recently, most of the vendor specific logic has been developed [in-tree](https://github.com/gardener/gardener). However, the project has grown to a size where it is very hard to extend, maintain, and test. With [GEP-1](https://github.com/gardener/gardener/blob/master/docs/proposals/01-extensibility.md) we have proposed how the architecture can be changed in a way to support external controllers that contain their very own vendor specifics. This way, we can keep Gardener core clean and independent.
+
+## Components
+Certificate services for Shoot clusters are provided by the composition of two components:
+
+- [Cert-Manager](https://github.com/jetstack/cert-manager)
+- [Cert-Broker](https://github.com/gardener/cert-broker)
+
+## Configuration
+Example configuration which is reconciled by this extension controller:
+```yaml
+apiVersion: certificateservice.extensions.gardener.cloud/v1alpha1
```

Comment:
WDYT about naming it `certificate-service.extensions.gardener.cloud/v1alpha1` for better readability?

Reply:
If I recall correctly there was a bug in code generator and having a dash in the last DNS label causes an error

Reply:
I tried it out, the generation also works with the dash.